### DISCUSSION TITLE
❓  Custom voice for German

### DISCUSSION THREAD

**Discussion by ChrisDelClea** [snakers4/silero-models#69] [Q&A] [answered]
## ❓ Questions and Help

Hi @snakers4,

great package! Is it possible to add a custom voice for German and if so, how?

Best regards 
Chris

**Comment by snakers4** [comment, marked as answer]
Hi,

Now a voice requires about 10-15-20 hours of recorded speech (see [this](https://github.com/thorstenMueller/deep-learning-german-tts/) for example).

We will be working on reducing this in the next iteration, but I doubt that the minimal amount will be less than 3-5 hours.

If you are interested in recording your voice  / custom voice in German - I guess you can just use Thorsten as a reference.

If you will choose to publish this voice and / or share it privately with us - we can add a new voice to the project.

**Reply by ChrisDelClea** [reply]
Hi @snakers4 ,

thanks for fast response. 
I think I could arrange 10h of recorded speech even though the background might be a bit noisy. 
3-5 ofc would be far easier! It's for an internal project in my company so I am not sure but can ask. 
Do you have a first entry point of how Thorsten did it? 

Best regards 
Chris

**Reply by snakers4** [reply]
> though the background might be a bit noisy.

Typically TTS requires no noise in the background, otherwise the quality degrades.
Of course you can use denoising, but usually it is just easier to invest a bit more in the recording (US$100 microphone).
Our speakers recorded in the studio (Ksenia) and at home (Aidar and Baya) and I cannot really say that the studio voice is better, though people recording at home did it in a quiet room with microphone with a pop-filter.

> Do you have a first entry point of how Thorsten did it?

I guess you can ask him directly @thorstenMueller (he is also active in his own repo mentioned above).
Also note that his dataset contains no stress, which ideally improves quality even further.

> It's for an internal project in my company

Typically we discuss commercial inquiries in dm, please reach out to hello@silero.ai or to @snakers41 in telegram
